### ISSUE TITLE
K2 Production Cluster Networking(?) Bug

### ISSUE THREAD

**Issue by leahnp** [samsung-cnct/k2#464]
Seeing a few pods in the production cluster "not being able to connect to host" in some capacity, here are the errors I see from failing/crashing pods:

```
ZOOKEEPER: 2017-05-30 23:15:16,171 [myid:3] - WARN  [QuorumPeer[myid=3]/0:0:0:0:0:0:0:0:2181:QuorumCnxManager@384] - Cannot open channel to 2 at election address zk-1.zk.kube-system.svc.cluster.local/IP
java.net.NoRouteToHostException: No route to host (Host unreachable)

TILLER: Cannot initialize Kubernetes connection: Get https://IP/api: dial tcp IP: getsockopt: no route to hostpanic: runtime error: invalid memory address or nil pointer dereference

FluentD: 2017-05-30 20:35:40 +0000 [error]: unexpected error error="Failed to open TCP connection to IP (Host is unreachable - connect(2) for \"IP" port 443)"

Fluent-Bit: Failed to start Sarama producer: kafka: client has run out of available brokers to talk to (Is your cluster reachable?)```

**Comment by alejandroEsc** [comment]
Is kubernetes having trouble routing to the other nodes/pods where this is coming from? this is a guess, but i would check to see if zk is set up properly?

**Comment by coffeepac** [comment]
this needs to move to the issues repo

**Comment by coffeepac** [comment]
closed in favor of samsung-cnct/issues#18
